api/util/translateDatetime.js

```diff
@@ -3,6 +3,7 @@ const moment = require('moment-timezone');
 
 module.exports = (input, timezone) => {
   const timeInZone = moment(new Date()).tz(timezone);
+  // TODO: This currently throws a moment error. Fix it?
```

Comment:
if the package was updated you can just lock it at the passing version.
you check their github issues tab to see any other users who were able to replicate or fix this?

Reply:
Don't think this is due to a recent package update because the error's been getting thrown for a while now.